src/components/Breadcrumb.vue

```diff
@@ -1,10 +1,12 @@
 <script>
+import getLocationValidator from '../utils/getLocationValidator';
+
 export default {
   props: {
     location: {
       type: Object,
       required: true,
-      validator: val => val._modelType && val._id,
+      validator: getLocationValidator(true),
```

Comment:
I'm confused somewhat by this line.  Should I be able to set `noRoot: true` but still allow a location with root to validate?

Reply:
This is something Vue prop validator won't able to do. Vue validator cannot take other props as part of the input. So, I validate allowing root here and later validate in connection with the no-root prop here. https://github.com/girder/girder_web_components/blob/browse_user-folder_collection_with_data-browser/src/components/Breadcrumb.vue#L86-L88